haystack/pipelines/standard_pipelines.py

```diff
@@ -714,9 +714,15 @@ def __init__(self, document_store: BaseDocumentStore):
         self.pipeline.add_node(component=document_store, name="DocumentStore", inputs=["Query"])
         self.document_store = document_store
 
-    def run(self, document_ids: List[str], top_k: int = 5):
+    def run(
+        self,
+        document_ids: List[str],
+        filters: Optional[Dict[str, Union[Dict, List, str, int, float, bool]]] = None,
```

Comment:
While we are at it, could we also include the option `index: Optional[str] = None` to the `run` and `run_batch` methods. I could see that option being useful to users.

Reply:
Done